src/main/groovy/com/marklogic/gradle/MarkLogicPlugin.groovy

```diff
@@ -121,7 +122,7 @@ class MarkLogicPlugin implements Plugin<Project> {
         project.task("mlUndeployApp", type: UndeployAppTask)
 
         String deployGroup = "ml-gradle Deploy"
-        project.task("mlPostDeploy", group: deployGroup, description: "Add dependsOn to this task to add tasks at the end of mlDeploy").mustRunAfter(["mlDeployApp"])
+        project.task("mlPostDeploy", group: deployGroup, dependsOn: "mlLoadSchemasData", description: "Add dependsOn to this task to add tasks at the end of mlDeploy").mustRunAfter(["mlDeployApp"])
```

Comment:
Instead of add this dependency, just add an instance of LoadSchemasDataCommand to the SimpleAppDeployer instance. Look around line 320 for how it's done for LoadModulesCommand. That command is also put into List that is added to the project, which allows for LoadModulesTask to fetch the already-constructed command instead of having to build the command itself. LoadSchemasDataTask can do likewise.